dashboard/client/index.js

```diff
@@ -41,8 +41,18 @@ Template.dashboard.onCreated(function () {
     }
   }
 
-  instance.getAggrData = opts => new Promise((resolve, reject) => {
-    Meteor.call('getAggr', opts, (err, res) => {
+  instance.getChartData = opts => new Promise((resolve, reject) => {
+    Meteor.call('getAggregatedData', opts, (err, res) => {
+      if (err) reject(err)
+      Meteor.call('parseDataForNvd', res, (err1, res1) => {
```

Comment:
I think we should reduce the number of round trips. If we want to decrease the client load by performing a second aggregation on the server, then send the data only **after** it has been aggregated properly.